src/components/graph/graph.renderer.jsx

```diff
@@ -105,7 +105,17 @@ function _renderDefs() {
             return cachedDefs;
         }
 
-        const small = MARKER_SMALL_SIZE;
+        //NOTE: this is a step in trying to offset the arrow placement based on the size
+        // of the Node.  Because these defs are static if the node sizes change
+        // the arrow location will still be in the same spot.  Also if the node is
+        // rectangular or irregular sometimes it will be off.
+        const maxSide = Math.max(config.node.width, config.node.height) / 10 / 2 || config.node.size / 10 / 2;
+        const minSide = Math.min(config.node.width, config.node.height) / 10 / 2 || config.node.size / 10 / 2;
```

Comment:
Maybe it would be a more clear naming `minSize`. What do you think?

Reply:
fixed